requirements.txt

```
@@ -1,6 +1,6 @@
-docutils==0.16
+docutils==0.17
 Sphinx>=3.4.2
```

So Sphinx 6 dropped docutils 0.17 support and below.
https://www.sphinx-doc.org/en/master/changes.html#release-6-0-0-released-dec-29-2022

So we can set a max and not install past 5.3.0 which is the version right before it on that same page.
```suggestion
Sphinx>=3.4.2,<=5.3.0
```